### PR TITLE
CIV-13859 Request for Reconsideration

### PR DESCRIPTION
**Pull request by miguelMolina3691**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-13859

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by laarmada — APPROVED** [review]

**Review by laarmada — APPROVED** [review]

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-CP-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "requestForReviewCommentsClaimant",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "CITIZEN-CLAIMANT-PROFILE",
+          "CITIZEN-DEFENDANT-PROFILE",
```
Comment (m-meulendijks-v1) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-CP-nonprod.json:9]: Defendant should probably just have read access, right?

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-CP-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      }
+    ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "requestForReviewCommentsDefendant",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "CITIZEN-CLAIMANT-PROFILE",
```
Comment (m-meulendijks-v1) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-CP-nonprod.json:22]: Claimant should probably just have read access, right?

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-CP-nonprod.json (hunk 1 above)
Comment (miguelMolina3691) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-CP-nonprod.json:9]: yes, it is more precise, I will modify it

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-CP-nonprod.json (hunk 2 above)
Comment (miguelMolina3691) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-CP-nonprod.json:22]: yes, it is more precise, I will modify it

**Review by m-meulendijks-v1 — APPROVED** [review]

**Pull request closed without merging**